debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/JavaVersionHelper.java

```diff
@@ -53,4 +52,22 @@ public static boolean ensureJava9AndUp(J9JavaVMPointer vm, PrintStream out) thro
 		}
 		return true;
 	}
+
+	/**
+	 * Returns true if the Java version is Java19 and up.
+	 * @param vm J9JavaVMPointer
+	 * @param out The output print stream
+	 * @return if the Java version is Java19 and up or not.
+	 * @throws CorruptDataException
+	 */
+	public static boolean ensureJava19AndUp(J9JavaVMPointer vm, PrintStream out) throws CorruptDataException
```

Comment:
It might be helpful to generalize this function so it receives the minimum required Java version.

Reply:
There are over 40 references to `ensureJava9AndUp` so we have to change all of them if we make it generic. Maybe we can do that in the future?

Reply:
Sure, replacing the uses of `ensureJava9AndUp()` can wait, but that doesn't mean we can't improve this now.

Reply:
Sure I can do that, is`ensureJavaVersion` a good name?

Reply:
I think the name should make it clear the version is a minimum, so I suggest `ensureMinimumJavaVersion()` or something like that.